app/models/solidus_subscriptions/subscription_generator.rb

```diff
@@ -56,5 +62,18 @@ def subscription_configuration(subscription_line_item)
         subscription_line_item.end_date
       )
     end
+
+    def find_or_create_stub_user(order)
+      existing_user = Spree.user_class.where(email: order.email).first
+
+      random_password = '14378417384'
```

Comment:
we can use `Devise.friendly_token` here to keep passwords, even if it's a spoofed registration, out of version control

Reply:
Are we risking exposing a default payment method on any of these accounts? We are basically creating an account without asking the user, should we send some kind of follow up email to let the claim the account?

Reply:
That's a good question, and I don't think we are exposing anything in particular here. We had talked about including a "get your account" link in the order confirm email and that's an option. Otherwise, in order to gain control of the account an attacker would need to have access to the email account, which is probably more valuable anyway.